terraform/helm/monitoring/files/rules/alerts.yml

```diff
@@ -30,7 +30,14 @@ groups:
     for: 5m
     labels:
       severity: error
-      summary: "State sync is not making progress (i.e., it is not keeping up with the head of the blockchain)"
+      summary: "State sync is not making progress (i.e., the synced version is not increasing!)"
+    annotations:
+  - alert: State sync is lagging significantly
+    expr: (aptos_data_client_highest_advertised_data{data_type="transactions"} - on(kubernetes_pod_name, role) aptos_state_sync_version{type="synced"}) > 1000000
```

Comment:
why this is only 1% of num_versions_to_skip_snapshot_sync?

Reply:
Aah, these two things are orthogonal:
1. The alert checks state sync lag in general, i.e., 1 million versions.
2. `num_versions_to_skip_snapshot_sync` is only used for fast sync to prevent data gaps, e.g., a node syncs to a version `Y`, crashes and then restarts with fast sync enabled. In this case, if the latest version of the blockchain is `X`, state sync will check if `X-Y <= num_versions_to_skip_snapshot_sync`. If that's the case, state sync will skip the snapshot sync and sync normally (e.g., continuous sync). But, if the gap is larger than `num_versions_to_skip_snapshot_sync` the node will panic with the error message telling the operator to change syncing modes or wipe the db.

Reply:
A newly bootstrapped node will certainly fire this, rigth? But seems fine to me.

Reply:
Yeah, a node coming online for the first time will fire this, but I guess that's okay? We can see how noisy it is and if it's too much, I can adjust it 😄

Reply:
@JoshLind - n00b question - can you clarify if this alert is specific to our nodes or it will fire for community nodes (atleast for validators) ?

Reply:
@sitalkedia I think these alerts will only fire for people who explicitly set them up. As far I know, only we do this for devnet, testnet, etc. So no one else will see these today 😊

Reply:
@JoshLind - In addition to individual node alerts, we should have high-level alerts as well, which define on an aggregate the health of the network?